src/main/java/net/sf/log4jdbc/StatementSpy.java

```diff
@@ -51,8 +70,7 @@ public class StatementSpy implements Statement, Spy
    *
    * @return the real Statement that this StatementSpy wraps.
    */
-  public Statement getRealStatement()
-  {
+  public Statement getRealStatement() {

```

Comment:
nit: Can we get rid of the style differences introduced probably by the IDE? Just to reduce noise.

**`[not-an-action-item]`** **`[soft-suggestion]`**

Reply:
Yes, I also noticed this, originally it was formatted in "more lines, more money" style. So, I'll need to check my changes and do the same.